great_expectations_cloud/agent/message_service/subscriber.py

```diff
@@ -95,9 +96,13 @@ def consume(
                 self.client.stop()
                 reconnect_delay = self._get_reconnect_delay()
                 time.sleep(reconnect_delay)  # todo: update this blocking call to asyncio.sleep
+                raise
             except KeyboardInterrupt as e:
                 self.client.stop()
                 raise KeyboardInterrupt from e
+            except GXAgentUnrecoverableConnectionError as e:
+                self.client.stop()
+                raise GXAgentUnrecoverableConnectionError from e
```

Comment:
It seems like you want to reraise the same error so this can just be `raise`.
```suggestion
                raise
```